task1/Program.cs

```diff
@@ -45,5 +56,175 @@ public static decimal PerformCalculation(int r)
+            }
+            Console.WriteLine($"Symbol {symbol} is met {count} times");
+        }
+
+        public static void InsertString()
+        {
+            Console.WriteLine("Enter text");
+            string text = Console.ReadLine();
+            Console.WriteLine("Enter inserted text");
+            string insertedText = Console.ReadLine();
+            Console.WriteLine("In what position should text be inserted?");
+            int position = Convert.ToInt32(Console.ReadLine());
+            string result = text.Insert(position, insertedText);
+            Console.WriteLine(result);
+        }
+
+        public static void DeleteSpaces()
+        {
+            Console.WriteLine("Input text");
+            string text = Console.ReadLine();
+            //string result = string.Empty;
+
+            int index1 = text.IndexOf('?', 0);
+            int index2 = text.IndexOf('?', index1 + 1);
+            char[] result = new char[text.Length];
+            StringBuilder str = new StringBuilder();
+            str = str.Append(text);
+            str.Replace(" ", null, index1, index2 - index1);
+            Console.WriteLine(str);
+
+        }
+
+        public static void DeleteExcessSpaces()
+        {
+            Console.WriteLine("Input text");
+            string text = Console.ReadLine();
+            string[] array = text.Split(' ');
+            foreach (string item in array)
+            {
+                item.Trim();
+            }
+            foreach (string item in array)
+            {
+                if (item == string.Empty)
+                    continue;
+                else
+                    Console.Write(item + " ");
+            }
+        }
+
+        public static void FirstSymbolOfWordByNumber()
+        {
+            Console.WriteLine("Input text");
+            string text = Console.ReadLine();
+            string[] array = text.Split(' ');
+            Console.WriteLine("Input number of word");
+            int position = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine(array[position - 1][0]);
+
+        }
+
+        public static void TransformToUpperCase()
+        {
+            Console.WriteLine("Input text");
+            string text = Console.ReadLine();
+            string[] array = text.Split(' ');
+            for (int i = 0; i < array.Length; i++)
+            {
+                array[i] = array[i][0].ToString().ToUpper() + array[i].Substring(1);
+            }
+            Console.WriteLine(string.Join(" ", array));
+        }
+
+        public static void ErrorPrint()
+        {
+            Console.WriteLine("Input printer string");
+            string text = Console.ReadLine();
+            int count = 0;
+            int countError = 0;
+            for (int i = 0; i < text.Length; i++)
+            {
+                if (text[i] == 'a' || text[i] == 'b' || text[i] == 'c' || text[i] == 'd' || text[i] == 'e' || text[i] == 'f' || text[i] == 'g' || text[i] == 'h' ||
```

Comment:
I see you spent a lot of time for that logic.
But there is a simpler way to do that:
the char 'm' is bigger than all previous chars.
So you could use '>' to define chars :).